cmd/podman/pod_restart.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	podRestartFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "restart all pods",
+		},
+		LatestFlag,
+	}
+	podRestartDescription = `Restarts one or more pods. The pod ID or name can be used.`
+
+	podRestartCommand = cli.Command{
+		Name:                   "restart",
+		Usage:                  "Restart one or more pods",
+		Description:            podRestartDescription,
+		Flags:                  podRestartFlags,
+		Action:                 podRestartCmd,
+		ArgsUsage:              "CONTAINER [CONTAINER ...]",
+		UseShortOptionHandling: true,
+	}
+)
+
+func podRestartCmd(c *cli.Context) error {
+	if err := checkMutuallyExclusiveFlags(c); err != nil {
+		return err
+	}
+
+	runtime, err := libpodruntime.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get runtime")
+	}
+	defer runtime.Shutdown(false)
+
+	args := c.Args()
+	var pods []*libpod.Pod
+	var lastError error
+
+	if c.Bool("all") {
```

Comment:
We do lines 49 through 74 in a lot of different pod related code.  Might consider moving it to a GetPods() type of utility function.

Reply:
@TomSweeneyRedHat I was going to wait till all of these merge, instead of having a separate PR for the commands that are in already, that would block what I have now. If that sounds ok to you.

Reply:
A separate cleanup PR would be fine by me.  I'd just ask that you make a trello card or a podman issue to track it.